### DISCUSSION TITLE
Mac OS Desktop App installable via Homebrew?

### DISCUSSION THREAD

**Discussion by MaxValue** [outline/outline#5258] [Ideas]
As of recently there seems to be a desktop app for Outline. But it can only be downloaded via the website.

It would be really convenient if it was installable via [homebrew](https://brew.sh/), because infrastructure-as-code and so on.

[Helpful Link](https://docs.brew.sh/Adding-Software-to-Homebrew)

**Comment by Djiit** [comment]
Would love to see this ! Any way we can contribute ?
